lib/classes/Service.js

```diff
@@ -61,65 +62,79 @@ class Service {
       serviceFilenames[serviceFileIndex] :
       _.first(serviceFilenames);
 
+    if (serviceFilename === 'serverless.js') {
+      return new BbPromise((resolve) => {
```

Comment:
Do not use the `new Promise` anti-pattern! Additionally require can throw synchronously, so it has to be catched.
```
return BbPromise.try(() => that.loadServiceFileParam(serviceFilename, require(serviceFilePath)));
```